components/callbacks/tasks.go

```diff
@@ -45,7 +45,7 @@ func (InvocationTask) Type() string {
 }
 
 func (t InvocationTask) Kind() hsm.TaskKind {
-	return hsm.TaskKindOutbound{Destination: t.Destination}
+	return hsm.TaskKindOutbound{Destination: t.BaseURL}
```

Comment:
Do we expect to have other kinds of destinations in the future? I think it's slightly confusing if a destination is always a URL to have two different terms that mean the same thing.